uber/templates/preregistration/confirm.html

```diff
@@ -20,6 +20,10 @@
             }, 1);
         {% endif %}
     });
```

Comment:
I'm guessing this was something you originally intended to use but then later didn't go with?  You should delete it if so.